### PR TITLE
zh-cn: update the translation of CRLF

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of CRLF

### CODE REVIEW

File: files/zh-cn/glossary/crlf/index.md [hunk 1]
```diff
@@ -5,16 +5,14 @@ slug: Glossary/CRLF
 
 {{GlossarySidebar}}
 
-回车符（CR）和换行符（LF）是文本文件用于标记换行的控制字符（[control characters](https://en.wikipedia.org/wiki/Control_character)）或字节码（[bytecode](https://en.wikipedia.org/wiki/Bytecode)）。
+回车符（CR）和换行符（LF）是文本文件用于标记换行的[控制字符](https://zh.wikipedia.org/wiki/控制字符)或[字节码](https://zh.wikipedia.org/wiki/字节码)。
 
-- CR = **Carriage Return**，回车符号（`\r`，十六进制 ascii 码为`0x0D`，十进制 ascii 码为 13），用于将鼠标移动到行首，并不前进至下一行。
-- LF = **Line Feed**，换行符号（ `\n`, 十六进制 ascii 码为 `0x0A`，十进制 ascii 码为 10）。
+- CR = **Carriage Return**，回车符号（`\r`，十六进制为`0x0D`，十进制为 13），用于将光标移动到行首，并不前进至下一行。
```
Comment (yin1999) [inline files/zh-cn/glossary/crlf/index.md:10]:
```suggestion
- CR = **Carriage Return**，回车符号（`\r`，十六进制为 `0x0D`，十进制为 13），用于将光标移动到行首，并不前进至下一行。
```

File: files/zh-cn/glossary/crlf/index.md [hunk 2]
```diff
@@ -5,16 +5,14 @@ slug: Glossary/CRLF
...
-回车符（CR）和换行符（LF）是文本文件用于标记换行的控制字符（[control characters](https://en.wikipedia.org/wiki/Control_character)）或字节码（[bytecode](https://en.wikipedia.org/wiki/Bytecode)）。
+回车符（CR）和换行符（LF）是文本文件用于标记换行的[控制字符](https://zh.wikipedia.org/wiki/控制字符)或[字节码](https://zh.wikipedia.org/wiki/字节码)。
 
-- CR = **Carriage Return**，回车符号（`\r`，十六进制 ascii 码为`0x0D`，十进制 ascii 码为 13），用于将鼠标移动到行首，并不前进至下一行。
-- LF = **Line Feed**，换行符号（ `\n`, 十六进制 ascii 码为 `0x0A`，十进制 ascii 码为 10）。
+- CR = **Carriage Return**，回车符号（`\r`，十六进制为 `0x0D`，十进制为 13），用于将光标移动到行首，并不前进至下一行。
+- LF = **Line Feed**，换行符号（`\n`, 十六进制为 `0x0A`，十进制为 10）。
 
-紧邻的 CR 和 LF（组成 CRLF，`\r\n`，或十六进制 `0x0D0A`）将鼠标移动到下一行行首。（Windows 操作系统默认的文本换行符为 CRLF；Linux 以及 macOS 系统默认使用 LF，早期的 mac os 系统使用 CR 换行。）
+紧邻的 CR 和 LF（组成 CRLF，`\r\n`，或十六进制 `0x0D0A`）将光标移动到行首，然后再移动至下一行。
```
Comment (yin1999) [inline files/zh-cn/glossary/crlf/index.md:13]: Note the order

Comment (yin1999) [reply files/zh-cn/glossary/crlf/index.md:13]: upstream: mdn/content#31055

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**